website/pages/en/querying/querying-by-subgraph-id-vs-deployment-id.mdx

```diff
@@ -4,32 +4,26 @@ title: Subgraph ID vs Deployment ID
 
 A subgraph is identified by a Subgraph ID, and each version of the subgraph is identified by a Deployment ID.
 
-When querying a subgraph, either ID can be used. However, there are some important differences between the two options.
+When querying a subgraph, either ID can be used, though it is generally suggested that the Deployment ID is used due to its ability to specify a specific version of a subgraph.
 
-![](/img/subgraph-id-vs-deployment-id.png)
-
-## Subgraph ID
-
-The Subgraph ID is a unique identifier for a subgraph. It remains constant across all versions of a subgraph. When queries are made using the Subgraph ID, you are not specifying a version. This means that the version of the subgraph that responds to your query may not be the latest version. Rather, it will be the most recent version that is synced and healthy.
-
-Example endpoint that uses Subgraph ID:
-
-`https://gateway-arbitrum.network.thegraph.com/api/[api-key]/subgraphs/id/FL3ePDCBbShPvfRJTaSCNnehiqxsPHzpLud6CpbHoeKW`
+Here are some key differences between the two IDs: ![](/img/subgraph-id-vs-deployment-id.png)
 
 ## Deployment ID
 
-When you're viewing a deployed subgraph, previously deployed versions are available for selection. Each version in that list has a unique Deployment ID.
+The Deployment ID is the IPFS hash of the compiled manifest file, which refers to other files on IPFS instead of relative URLs on the computer. For example, the compiled manifest can be accessed via: `https://api.thegraph.com/ipfs/api/v0/cat?arg=QmQKXcNQQRdUvNRMGJiE2idoTu9fo5F5MRtKztH4WyKxED`. To change the Deployment ID, one can simply update the manifest file, such as modifying the description field as described in the [subgraph manifest documentation](https://github.com/graphprotocol/graph-node/blob/master/docs/subgraph-manifest.md#13-top-level-api).
 
-When you make queries using the Deployment ID, you are specifying a version of that subgraph to query. This means that the version of the subgraph that responds to your query will be the version specified by the Deployment ID.
+When queries are made using a subgraph's Deployment ID, we are specifying a specific version of that subgraph to query. Using the Deployment ID to query a specific subgraph version results in a more sophisticated and robust setup as there is full control over the subgraph version being queried. However, this results in the need of updating the query-code manually every time a new version of the subgraph is published.
```

Comment:
I would remove the first "specific" since it's a bit repetitive ("specifying a specific version"). Also, not sure why there's a dash in "query-code"?
```suggestion
When queries are made using a subgraph's Deployment ID, we are specifying a version of that subgraph to query. Using the Deployment ID to query a specific subgraph version results in a more sophisticated and robust setup as there is full control over the subgraph version being queried. However, this results in the need of updating the query code manually every time a new version of the subgraph is published.
```